src/controller/CommissionerDiscoveryController.cpp

```diff
@@ -230,21 +230,7 @@ void CommissionerDiscoveryController::InternalOk()
     {
         ChipLogDetail(AppServer, "UX InternalOk: app not installed.");
 
-        // notify client that app will be installed
-        CommissionerDeclaration cd;
-        cd.SetErrorCode(CommissionerDeclaration::CdError::kAppInstallConsentPending);
-        mUdcServer->SendCDCMessage(cd, Transport::PeerAddress::UDP(client->GetPeerAddress().GetIPAddress(), client->GetCdPort()));
-
-        // dialog
-        ChipLogDetail(Controller, "------PROMPT USER: %s is requesting to install app on this TV. vendorId=%d, productId=%d",
-                      client->GetDeviceName(), client->GetVendorId(), client->GetProductId());
-
-        if (mUserPrompter != nullptr)
-        {
-            mUserPrompter->PromptForAppInstallOKPermission(client->GetVendorId(), client->GetProductId(), client->GetDeviceName());
-        }
-        ChipLogDetail(Controller, "------Via Shell Enter: app install <pid> <vid>");
-        return;
+        // TODO: Prepare app to be installed or add it to the mContentApps
```

Comment:
Add issue number to TODO

Reply:
I have a follow up PR that is in draft mode right now:
https://github.com/project-chip/connectedhomeip/pull/33982/

Reply:
I can add that as a note

Reply:
The PR should be fixing an issue. Please make sure you have an issue that #33982 will fix. That way we can track end-to-end without needing to track draft PRs. The issue number should be "fixed" by 33982, but listhere in the TODO above.

Reply:
Internally decided to take another approach. I've removed TODO: